src/util/DomWrapper.ts

```diff
@@ -1,7 +1,7 @@
 import { WidgetBase } from './../WidgetBase';
-import { Constructor, DNode, VirtualDomProperties, WidgetProperties } from './../interfaces';
+import { Constructor, DNode, HNode, VirtualDomProperties, WidgetProperties } from './../interfaces';
 import { v } from './../d';
-import { VNode } from '@dojo/interfaces/vdom';
+import { InternalHNode } from './../vdom';
```

Comment:
We probably need to get rid of VNode in interfaces after we land this, right?  It is used in a couple places (like test-extras) which sort of had to deal with maquette nodes directly.

Reply:
Yup we can remove them afterwards for sure and yup test-extras is going to suffer the most from these changes. Otherwise updating widgets was just a matter of swapping `VNode`s for `DNodes`